dbt-metricflow/dbt_metricflow/cli/cli_link.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from typing import Optional
+
+import click
+from packaging.version import Version
+
+
+class CliLink:
+    """Groups methods for links / URLs that displayed through the CLI."""
+
+    @staticmethod
+    def _blue_text(url: str) -> str:
+        return click.style(url, fg="blue", bold=True)
+
+    @staticmethod
+    def get_time_spine_docs_link(dbt_core_version: Optional[Version]) -> str:  # noqa: D102
+        time_spine_docs_url = "https://docs.getdbt.com/docs/build/metricflow-time-spine"
```

Comment:
nit: might a good idea to leave all these URLs on the top as constants so it's all in one place

Reply:
Updated.